viz_scripts/generic_metrics_sensed.ipynb

```diff
@@ -166,30 +195,27 @@
    },
    "outputs": [],
    "source": [
-    "plot_title_no_quality=\"Miles for each primary mode\\n(inferred by OpenPATH from phone sensors)\"\n",
+    "plot_title_no_quality = label_units + \" for each primary mode\\n(inferred by OpenPATH from phone sensors)\"\n",
+    "plot_title = plot_title_no_quality + quality_text\n",
```

Comment:
I moved it back! I think I glossed over the difference between `plot_title` and `plot_title_no_quality` thinking they both could go outside the try block - I also don't see how this could fail, but better to be safe (and match the convention in the rest of the file)[e89be8a](https://github.com/e-mission/em-public-dashboard/pull/114/commits/e89be8af2f431d13e4d1e474bd703cfbd2551891)